src/components/chipsInput/index.js

```diff
@@ -292,7 +297,7 @@ class ChipsInput extends Component {
     return (
       <View style={styles.inputWrapper}>
         <TextField
-          ref={r => (this.input = r)}
+          ref={r => (this.state.inputRef = r)}
```

Comment:
You can use `React.createRef()` instead

Reply:
If I move it do didMount, it can stay the was it was

Reply:
Move what to `DidMount`?

Reply:
it's my bad! I meant `DidUpdate`

Reply:
You don't need to update the ref. You create it once in the c'tor: `this.input = React.createRef();`.